### ISSUE TITLE
mfs.cgi not show ip address in Metadata Servers tab.

### ISSUE THREAD

**Issue by andypl78** [lizardfs/lizardfs#246]
As the topic.

![noip](https://cloud.githubusercontent.com/assets/7813646/5878077/9204c33e-a324-11e4-96dd-1e9503aaee1f.jpg)

**Comment by marcinsulikowski** [comment]
There is a dash which indicates that we refuse to guess it :). We may place some address there, but in a HA solution which we are going to prepare for the next release most probably you would see a virtual, floating IP address (which does not identify any particular machine). In this table "host" is a real hostname used by `corosync` and `lizardfs-cluster-manager` and IP is the address of a network interface used to handle master<->shadow connection. We don't resolve any hostnames nor use any reverse lookup. We wanted to avoid showing a table like this:
```
#   host    ip             personality
1   node3   10.10.10.100   master
2   node1   10.10.10.1     shadow
3   node2   10.10.10.2     shadow
4   node4   10.10.10.4     shadow
```
where in `node3`'s ip column a virtual address is present. One may expect to see something consistent with the rest of nodes (10.10.10.3 in this example), but typically no IP connection uses this address.

To sum up: we had a choice between showing nothing (e.g., `-`) or showing some inconsistent data which would just fill the cell in this table. We have chosen the first option.

**Comment by andypl78** [comment]
Ok, thx for the explanation.

**Comment by onlyjob** [comment]
Unless I miss anything actionable I believe this bug is safe to close.
